packages/ember-toucan-core/src/components/form/field.hbs

```diff
@@ -5,6 +5,7 @@
       Hint=this.Hint
       Control=this.Control
       Error=this.Error
+      InlineError=this.InlineError
```

Comment:
I haven't really looked into what the differences for these two error components are, but have you looked into whether it's possible to have just one, and maybe if absolutely needed pass an argument to distinguish the two modes? I am not insisting here, as again I haven't looked deeply into it and don't know the tradeoffs exactly, just thinking aloud...

Reply:
Would something like what I described at https://github.com/CrowdStrike/ember-toucan-core/pull/115#discussion_r1164214606 work?  Still use the existing Error component, but conditionally apply additional classes (or even use a new component argument to apply the styles we want!)?

Reply:
InlineError is removed.